### DISCUSSION TITLE
Split now has standard 2 areas. Would be nice if I could choose between 1 to 6 areas

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#138] [Ideas]
hi Renaud,

Split now has standard 2 areas. Would be nice if I could choose between 1 to 6 areas.

kind regards
Folkert

**Comment by renaudpawlak** [comment]
Indeed it could be convenient, but I want to think about the global picture. There are various types of possible layouts and some are still missing, but they are all possible using CSS and sub-containers. For instance, it is easy to split with 4 area (1 horizontal split and then 2 vertical splits on the left and right areas). Add another vertical split and you get a 6-area container, and so on.

So, at that point, I don't want to multiply too much the components and avoid configuration that could be done in multiple ways.

Also, I believe that a more urgent feature would be a way to switch between various types of containers and wrap existing components in new containers easily (right now it is too complicated).

Anyways, in the future "simple" version, that type of components could be useful for users wanting to get thing done fast without minimalistic configuration. I will keep it in mind when I start working on the "simple" version.
